.github/workflows/eden_gcp.yml

```diff
@@ -152,8 +152,8 @@ jobs:
         run: |
           if [ "${{ github.event.inputs.test_type }}" == "PR" ]; then
             git fetch origin pull/${{ github.event.inputs.pr_number }}/head && git checkout FETCH_HEAD
-            echo "EVE_TAG=pr${{ github.event.inputs.pr_number }}" >> "$GITHUB_ENV"
-            echo "EVE_REGISTRY=evebuild/danger" >> "$GITHUB_ENV"
+            echo "EVE_TAG=${{ github.event.inputs.pr_number }}" >> "$GITHUB_ENV"
```

Comment:
I don't think this will work, because it'll try to download it from docker registry, but we didn't do docker load here, but since this is only for manual runs I guess we can leave it be and rework it later